### PR TITLE
Nav bar styling

### PR DESCRIPTION
**Pull request by nmking22**
* Add image links for sign in, sign out and home to navbar
* Update flash message styling
* Adapt tests to work with new styling

- [x] Repo Representation: Front End

- [x] All Tests are passing

- [x] Updated README.md with added information - screenshots will need to be added for this after the welcome page styling is done

### CODE REVIEW

File: spec/features/welcome/landing_spec.rb [hunk 1]
```diff
@@ -44,17 +46,23 @@
       stub_omniauth
 
       visit root_path
-      within('.login-link') do
-        click_link 'Log In with Google'
+
+      within('.navbar-header') do
+        within('.sign-in-link') do
```
Comment (SageOfCode) [inline spec/features/welcome/landing_spec.rb:51]: Great work on these tests, I like this use of css.

**Review by SageOfCode — APPROVED** [review]
Looks good to me.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -14,20 +14,30 @@
...
+          <% unless current_user %>
+            <div class="sign-in-link">
+              <%= link_to image_tag("sign_in.png", size:100, class: 'sign-in'), google_login_path %>
+            </div>
+          <% else %>
+            <div class="sign-out-link">
+              <%= link_to image_tag("sign_out.png", size:100, class: 'sign-out'), google_logout_path %>
+            </div>
+          <% end %>
+        </div>
```
Comment (JesseMellinger) [inline app/views/layouts/application.html.erb:30]: Looking forward to seeing these images in the nav bar!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -14,20 +14,30 @@
...
-        <%= message %>
-      </h4>
-   <% end %>
+    <% flash.each do |type, message| %>
+      <div class="flash-messages">
+        <h4 class='<%= type %>'>
+          <%= message %>
+        </h4>
+      </div>
+    <% end %>
```
Comment (JesseMellinger) [inline app/views/layouts/application.html.erb:40]: Thanks for adding styling to the error messages.

**Review by JesseMellinger — APPROVED** [review]

**Review by GeorgieGirl24 — APPROVED** [review]

**Pull request merged**